### DISCUSSION TITLE
PlanetScale registering reads without any connection

### DISCUSSION THREAD

**Discussion by gabsdotco** [planetscale/discussion#173] [Bug Reports] [answered]
Currently I'm working on a database with 2 branches, one for the production (main) and one for development (dev), the dev branch seems to correctly register the reads and writes timeline, but the main branch keeps showing register at each minute. How is this possible?

**Main branch:**

![image](https://user-images.githubusercontent.com/41910417/155728335-8b6a6b23-1455-4b92-9a4f-60874d6d91d3.png)

**Dev branch:**

![image](https://user-images.githubusercontent.com/41910417/155728874-4e3bd6d6-6bd9-409f-bb2d-c2a78edef4ad.png)

**Comment by jgreet** [comment, marked as answer]
Sorry for the confusion here. This is the same situation as https://github.com/planetscale/beta/discussions/111. The regular reads on the the main branch area are coming from an internal health check.

**Reply by gabsdotco** [reply]
No problem, thought it was a problem from my side! Thank you for the answer!
